beproud/django/notify/tasks.py

```diff
@@ -4,41 +4,24 @@
 from django.conf import settings
 
 import celery
-from celery.task import Task
-from celery.registry import tasks
-
-if celery.VERSION < (3, 1):
-    try:
-        import djcelery  # NOQA
-    except ImportError:
-        from django.core.exceptions import ImproperlyConfigured
-        raise ImproperlyConfigured("when used celery<3.1, djcelery is required!")
-
-    if 'djcelery' not in settings.INSTALLED_APPS:
-        from django.core.exceptions import ImproperlyConfigured
-        raise ImproperlyConfigured("djcelery not in INSTALLED_APPS!")
+from celery import shared_task
 
 from beproud.django.notify import notify_now
 
-
-class Notify(Task):
-
-    def run(self, targets, notify_type, extra_data={}, include_media=None, exclude_media=[],
+@shared_task(bind=True)
+def Notify(self, targets, notify_type, extra_data={}, include_media=None, exclude_media=[],
```

Comment:
MAY: 関数なら小文字にしてほしい

```suggestion
def notify(self, targets, notify_type, extra_data={}, include_media=None, exclude_media=[],
```

後方互換性維持のためであれば、以下の様に互換性のための別名定義をすればよさそうです。

```python
def notify(...):
    ...

Notify = notify  # backward-compatibility
```